src/services/severa-api-service.ts

```diff
@@ -33,200 +34,113 @@ export const CreateSeveraApiService = (): SeveraApiService => {
   return {
     /**
      * Gets flextime by severaUserId
-     * 
+     *
      * @param severaUserId  Severa user id
      * @returns Flextime of a user
      */
     getFlextimeBySeveraUserId: async (severaUserId: string) => {
-
       const eventDateYesterday = DateTime.now().minus({ days: 1 }).toISODate();
-  
+
       const url = `${baseUrl}/v1/users/${severaUserId}/flextime?eventdate=${eventDateYesterday}`;
 
       const response = await fetch(url, {
         method: "GET",
         headers: {
           Authorization: `Bearer ${await getSeveraAccessToken()}`,
-          "Client_Id": process.env.SEVERA_DEMO_CLIENT_ID,
-          "Content-Type": "application/json",
-        },
+          Client_Id: process.env.SEVERA_DEMO_CLIENT_ID,
+          "Content-Type": "application/json"
+        }
       });
 
       if (!response.ok) {
-        throw new Error(
-          `Failed to fetch flextime: ${response.status} - ${response.statusText}`,
-        );
+        throw new Error(`Failed to fetch flextime: ${response.status} - ${response.statusText}`);
       }
       return response.json();
     },
-      
-    /**
-    * Retrieves the test user information from environment variables.
-    * 
-    * @returns An object containing the test user's email and GUID (ID).
-    * @throws Error if the required environment variables (SEVERA_TEST_USER_ID or SEVERA_TEST_USER_EMAIL) are not set.
-    */
-    getTestUser : async ()  => {
-      if (!process.env.SEVERA_TEST_USER_ID|| !process.env.SEVERA_TEST_USER_EMAIL) {
-        throw new Error("SEVERA_TEST_USER_EMAIL environment variable is missing.");
-      }
-      return {
-      email: process.env.SEVERA_TEST_USER_EMAIL,
-      guid: process.env.SEVERA_TEST_USER_ID 
-      };
-    },
 
     /**
-    * Fetches a Severa user by their email address and checks/updates the 'isSeveraOptIn' status.
-    * 
-    * @param email The email address of the user to be retrieved.
-    * @param attribute Attributes to be checked/updated for the user.
-    * 
-    * @returns The user's Severa GUID, 'isSeveraOptIn' status, and email.
-    */
-    getUserByEmail: async (email: string, attribute: Record<string, string[]>) => {
+     * Fetches a Severa user by their email address and checks/updates the 'isSeveraOptIn' status.
+     *
+     * @param email The email address of the user to be retrieved.
+     * @param keyword Keywords to be checked/updated for the user.
+     *
+     * @returns The user's Severa GUID, 'isSeveraOptIn' status, and email.
+     */
+    getUserByEmail: async (email: string, keyword: Record<string, string[]>) => {
+      dotenv.config();
+      const isLocal = process.env.STAGE === "local";
+
+      const testEmail = isLocal ? Config.get().testUser.email : email;
```

Comment:
If this is handling the develop environemnt "test email" and the actual user provided email as it seems to be, then this should be renamed to email, or as we already have the email param, i guess, "userEmail"